Sticky-Column-Cards

altugoz opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #23

Test URLs:
- Before: https://main--wbcopy--helms-charity.aem.page
- After: https://23-sticky-columns-cards--wbcopy--helms-charity.aem.page

altugoz committed:
26

altugoz committed:
Changed link color to white

altugoz committed:
Cards background color changed

iustinp reviewed (changes requested):
There are a few things:
0. The use of !important throughout the code is a bad practice. !important should only be used when there is no possible alternative. If you really feel the need to use it, let's talk about it, or call the !important Hotline, where someone is always ready to help you say NO! to !Important! XD
 
1. cards are overall smaller than on the original site
2. gap between them is therefore much larger too. This affects most of the other things:
3. Height and width of the image are different, and also the ratio between height and width is different. Fixing hight and width will also fix ratio.
4. Same for text area.
5. Title text on cards is smaller than original
6. Hover color of text area is different. The color code for anything needs to be taken directly from the original site, and not adjusted only by visual approximation
7. Bottom distance for  IBRD - IDA MIGA IFC links is larger than for the rest of the links
8. Underline thickness is different for all links than on the original
9. Color for underlines should be var(--text-on-light-link-hover), which in turn is: --text-on-light-link-hover: var(--blue-50-base); which in turn is     --blue-50-base: #0071bc;. You are now using var(--accent-blue-primary-40);. Since all these variables have been brought with wpcopy from the original, it's better to use them as we find them in the original. 
10. The main title of the cards is centerd left instead of center
11. The "New for 2025" eyebrow on the first card is missing.

altugoz committed:
feat: implement sticky columns card layout matching World Bank Academy

- Added 1-left, 2-right sticky column layout with CSS Grid
- Matched exact dimensions (646x364px images, World Bank padding)
- Applied dark blue background (#001c2d) and light blue-gray section bg
- Styled links with proper underlines and hover effects
- Added CSS-generated 'New for 2025' badge on first card
- Centered main section title
- Set consistent spacing (1.5rem gap) between cards
- Made all text, headers, and links match World Bank Academy styling
- Removed all !important declarations for cleaner CSS

altugoz committed:
1. Gap between sticky card and right card fixed
2. Mobile and Tablet view is fixed

altugoz committed:
1. Overlap of cards fixed
2. Underline color fixed
3. Background color fixed from hexadecimal to variable

altugoz committed:
Fix sticky columns card styling

altugoz committed:
1. Cards size modified as per original ones
2. Underline hover color changed
3. Padding on breakpoints modified

iustinp committed:
Merge branch 'main' into 23-sticky-columns-cards
